server/resolvers/book/query.js

```diff
@@ -0,0 +1,90 @@
+const { Op } = require("sequelize");
+const { Author, Book } = require("../../models");
+const { Metadata } = require("../../models/mongodb");
+
+const BookQuery = {
+  books: async ({ page = 1, limit = 10, filter = {} }) => {
+    const offset = (page - 1) * limit;
```

Comment:
the pagination part seems like something which will apply to all queries, any way to make it common?